src/features/meeting-frame/meeting-frame.js

```diff
@@ -78,6 +78,8 @@ export default class MeetingFrame extends React.Component {
             'audioMuteStatusChanged', this._onAudioMuteChange);
         this._jitsiApi.addListener(
             'feedbackSubmitted', this.props.onMeetingLeave);
+        this._jitsiApi.addListener(
+            'proxyConnectionEvent', this._onSendMessageToRemoteControl);
```

Comment:
reading this, i thought it was the handler for an incoming (iframe host -> iframe) event, but it looks like this is an outgoing (iframe -> iframe host) event?